site/src/pages/CreateWorkspacePage/CreateWorkspacePageExperimental.tsx

```diff
@@ -32,6 +28,7 @@ import { paramsUsedToCreateWorkspace } from "utils/workspace";
 import { CreateWorkspacePageViewExperimental } from "./CreateWorkspacePageViewExperimental";
 export const createWorkspaceModes = ["form", "auto", "duplicate"] as const;
 export type CreateWorkspaceMode = (typeof createWorkspaceModes)[number];
+import { type CreateWSPermissions, createWorkspaceChecks } from "./permissions";
```

Comment:
```suggestion
import { type CreateWorkspacePermissions, createWorkspaceChecks } from "./permissions";
```

can we also rename this type while we're at it? I've always hated this name 😭